aptos-move/aptos-aggregator/src/bounded_math.rs

```diff
@@ -1,67 +1,12 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-use aptos_logger::error;
-use move_binary_format::errors::PartialVMError;
-use move_core_types::vm_status::StatusCode;
-
-// When bounded math operation overflows
-// Generally means addition exceeded limit.
-pub(crate) const EBOUND_OVERFLOW: u64 = 0x02_0001;
```

Comment:
Where are all the error codes now?

Reply:
only errors we throw (that get on chain) come from add/sub methods (not try_add/try_sub) from move code. 
errors are in aggregators_v2.move


    /// The value of aggregator overflows. Raised by uncoditional add() call
    const EAGGREGATOR_OVERFLOW: u64 = 1;

    /// The value of aggregator underflows (goes below zero). Raised by uncoditional sub() call
    const EAGGREGATOR_UNDERFLOW: u64 = 2;

    /// The generic type supplied to the aggregator snapshot is not supported.
    const EUNSUPPORTED_AGGREGATOR_SNAPSHOT_TYPE: u64 = 5;

    /// The aggregator snapshots feature flag is not enabled.
    const EAGGREGATOR_SNAPSHOTS_NOT_ENABLED: u64 = 6;

    /// The generic type supplied to the aggregator is not supported.
    const EUNSUPPORTED_AGGREGATOR_TYPE: u64 = 7;